### PR TITLE
Issue/325

### PR DESCRIPTION
**Pull request by jgostick**
This address issue #325 once and for all.  I think @TomTranter did a big batch of changes, but these catch the last few.

### CODE REVIEW

File: OpenPNM/Physics/__GenericPhysics__.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def __init__(self, network=None, phase=None, geometry=None,
             self._phases.append(phase)  # Register phase with self
 
         if geometry is not None:
-            if (pores != []) or (throats != []):
+            if (sp.size(pores) > 0) or (sp.size(throats) > 0):
```
Comment (bspellio) [inline OpenPNM/Physics/__GenericPhysics__.py]: We don't actually need these parens.

Comment (jgostick) [inline OpenPNM/Physics/__GenericPhysics__.py]: Let's keep'em, unless pep8 is being a little cry baby about them.  Bool
comparisons can behave very stragely without parens, in my experience.

Comment (bspellio) [inline OpenPNM/Physics/__GenericPhysics__.py]: :+1:

**Comment by bspellio** [comment]
:ship:

**Pull request merged**